lib/dash.coffee

```diff
@@ -86,7 +86,10 @@ plugin = module.exports =
     return keys.map(encodeURIComponent).join(',') if keys.length
 
   getDashURI: (string, path, language, background) ->
-    link = 'dash-plugin://query=' + encodeURIComponent(string)
+    if process.env.XDG_SESSION_DESKTOP == "KDE"
+      link = 'dash-plugin:query=' + encodeURIComponent(string) #plays nice with KDE Plasma 5
+    else link = 'dash-plugin//:query=' + encodeURIComponent(string)
```

Comment:
Just for the reference, my [comment](https://github.com/blakeembrey/atom-dash/issues/37#issuecomment-237140968) in #37 explains why removing double slash is a good idea for any Linux, not just KDE.